test/verify/check-static-login

```diff
@@ -1110,6 +1110,27 @@ matchrule = <SUBJECT>^DC=LAN,DC=COCKPIT,CN=alice$
         check_session()
         check_store_hostkey(real_hostkey)
 
+    @testlib.skipOstree("No NFS server")
+    def testNFSHomeDir(self):
```

Comment:
As this only affects SELinux do we want to whitelist this Fedora/RHEL/CentOS which have SELinux?

Reply:
Well, there could be other things that break it -- AppArmor, seccomp, or bugs in other places. I think it doesn't hurt to just cover it everywhere.